### PR TITLE
Documentation part 5 building block view

### PR DESCRIPTION
**Pull request by luisfesu**

### CODE REVIEW

File: webapp/docs/05_building_block_view.adoc [hunk 1]
```diff
@@ -56,19 +56,30 @@ Since there are so many ways to specify interfaces why do not provide a specific
...
 ****
+// _**<Overview Diagram>**_ //
 
-_**<Overview Diagram>**_
+image:Doc5-BuildingBlockView.png["Hierarchy of building blocks"]
 
 Motivation::
 
-_<text explanation>_
+The Radarin System accessed by two types of actors (users & administrator) which conects to the Solid POD Provider to access the user personal data.
```
Comment (ghost) [inline webapp/docs/05_building_block_view.adoc]: accessed -> is accessed

File: webapp/docs/05_building_block_view.adoc [hunk 2]
```diff
@@ -56,19 +56,30 @@ Since there are so many ways to specify interfaces why do not provide a specific
...
+The system will be subdived in two main software components to meet the project requirements.
```
Comment (ghost) [inline webapp/docs/05_building_block_view.adoc]: subdived in -> subdivided into

File: webapp/docs/05_building_block_view.adoc [hunk 3]
```diff
@@ -150,6 +161,25 @@ Please prefer relevance over completeness. Specify important, surprising, risky,
...
+Show the decomposition of the Radarin system in the two software components, how both of them relate with the different actors in the application and the way they interact with the User POD and other neighbour systems.
+
+
+Contained Building Blocks::
+[options="header",cols="1,3"]
+|===
+|**Name** | **Responsability**
+|Mobile Application | Used by users to recive notifications from nearby friends and sending locations to the server.
+
+|Web Application | Used by the users to manage and visualize locations and by the administrator to manage users and monitor server status. It access the user data by means of the POD.
```
Comment (ghost) [inline webapp/docs/05_building_block_view.adoc]: access -> accesses

File: webapp/docs/05_building_block_view.adoc [hunk 4]
```diff
@@ -150,6 +161,25 @@ Please prefer relevance over completeness. Specify important, surprising, risky,
...
+
+|User POD | Provided by the Solid POD Provider, stores the personal data of the user such as it friend list.
```
Comment (ghost) [inline webapp/docs/05_building_block_view.adoc]: it -> his/her

File: webapp/docs/05_building_block_view.adoc [hunk 5]
```diff
@@ -183,24 +213,19 @@ When you need more detailed levels of your architecture please copy this
...
-
-==== White Box <_building block x.2_>
+Show the decomposition of the Web Application to include relevant parts of the main part of the system, the web application.
 
-_<white box template>_
+Contained Building Blocks::
+[options="header",cols="1,3"]
+|===
+|**Name** | **Responsability**
+|Components | Javascript components created by React as a way of building blocks of code and ensure reusability and independence.
```
Comment (ghost) [inline webapp/docs/05_building_block_view.adoc]: Javascript -> JavaScript

File: webapp/docs/05_building_block_view.adoc [hunk 6]
```diff
@@ -183,24 +213,19 @@ When you need more detailed levels of your architecture please copy this
...
 
+|ComponentsTests | Tests included in the web application to ensure the proper functioning of the React components.
 
+|Services | Clases of the application in charge of interact with the Solid POD's and use the user data in a proper way. 
+|===
```
Comment (ghost) [inline webapp/docs/05_building_block_view.adoc:229]: interact -> interacting
use -> using

**Comment by ghost** [comment]
I have made some little changes to the .adoc, do you like it?

**Comment by luisfesu** [comment]
It's perfect, thanks for the contribution @uo271913 .

**Pull request merged**